Formula/ttfautohint.rb

```diff
@@ -41,7 +41,7 @@ def install
     args << "--without-qt" if build.without? "qt"
 
     system "./bootstrap" if build.head?
-    system "./configure", *args
+    system "QMAKESPEC='macx-clang' ./configure", *args
```

Comment:
what's the deal here?

Reply:
`configure` uses `autotroll.m4` to identify Qt5, which by default uses the 'macx-g++' spec for building.  However, this isn't supported by homebrew, thus it has to be overridden with the `QMAKESPEC` environment variable (as documented for `qmake`)

Reply:
Ah, sorry, I haven't seen your comment earlier.  Well, at least two days ago I got a report that it *is* necessary to set `QMAKESPEC` on current homebrew...  In case this hasn't be fixed recently, it should stay :-)